fdtd/fdtd1d.py

```diff
@@ -48,12 +48,12 @@ def computeRHSE(self, fields):
         elif self.mesh.boundary_label  == "Periodic":
             rhsE[1:] = - (1.0/self.dxH) * (H[1:] - H[:-1])
             rhsE[0] = - (1.0/self.dxH[0]) * (H[0] - H[-1])         
-            #rhsE[-1] = rhsE[0]
+            #rhsE[-1] = rhsE[0] ##
 
         elif self.mesh.boundary_label =="PMC":
             rhsE[1:-1] = - (1.0/self.dxH) * (H[1:] - H[:-1])
-            rhsE[0] = rhsE[0] - (2*H[0])
-            rhsE[-1] = rhsE[-1] - (-2*H[-1])
+            rhsE[0] = rhsE[0] - (1.0/self.dxH[0]) * (2 * H[0])
+            rhsE[-1] = rhsE[-1] - (1.0/self.dxH[0]) * (-2 * H[-1])
```

Comment:
This won't work with CFL < 1.0 add a test that checks pmc running until 1.0 with CFL 0.5

Reply:
Test is done. CFL values between 0 and 1 for PMC test seem to not alter expected H field behaviour. For a time of 1.0 wave hasn't traveled all the way back and cant check the reflected field condition for H, only that is null, about to be inverted.